mrp_operations_extension/models/mrp_production.py

```diff
@@ -157,3 +157,14 @@ def action_start_working(self):
         if self.production_id.state in ('confirmed', 'ready'):
             self.production_id.state = 'in_production'
         return super(MrpProductionWorkcenterLine, self).action_start_working()
+
+    def action_done(self):
+        for move_line in self.move_lines:
```

Comment:
We detected a bug on operations extension. If one operation is finished all pending moves have to be canceled. Allowing to consume or to produce after ending the operation was causing big troubles on standar MO workflow and it's not conceptually correct.
This fix is reparing the issue